wrap.go

```diff
@@ -136,6 +136,37 @@ func As[T Steper](s Steper) []T {
 	}
 }
 
+// IsStep reports whether there is any step matches target step.
+func IsStep(step, target Steper) bool {
+	if step == target {
+		return true
+	}
+	if step == nil || target == nil {
+		return false
+	}
+	for {
+		if step == target {
+			return true
+		}
+		switch u := step.(type) {
+		case interface{ Unwrap() Steper }:
+			step = u.Unwrap()
+			if step == nil {
+				return false
+			}
+		case interface{ Unwrap() []Steper }:
+			for _, s := range u.Unwrap() {
+				if IsStep(s, target) {
+					return true
```

Comment:
The func `IsStep` means that, `step` contains `target`?

Reply:
suppose step `A` warpping `B`, that's
```
func (a *A) Unwrap() flow.Steper { return &a.B }
```
Then workflow will only orchestrate `A` and ignore `B`,
so from the perspective of workflow, there is not `A` containing `B`, rather `A` is `B`.
However, this is kinda tricky and I just try to reuse the `Is` conception from `flow.Is` and `errors.Is`

Reply:
Understood, thanks for the explanation.